observations/obs_converters/utilities/create_sphere_obs.f90

```diff
@@ -110,18 +147,19 @@ program create_sphere_obs
    lon = atan2(y, x) + pi
    lat = asin(z)
    
-   dlon = rad2deg * lon
-   dlat = rad2deg * lat
+   deglon = rad2deg * lon
+   deglat = rad2deg * lat
 
    do lvl = 1, nlvls
-      call set_obs_def_location(obs_def, set_location(dlon, dlat, obs_levels(lvl), VERTISPRESSURE))
+
+      call set_obs_def_location(obs_def, set_location(deglon, deglat, obs_levels(lvl)*100.0_r8, VERTISPRESSURE))
    
-      call set_obs_def_error_variance(obs_def, 1.0_r8)
+      call set_obs_def_error_variance(obs_def, rawin_temp_error(obs_levels(lvl)))
       call set_obs_def_type_of_obs(obs_def, RADIOSONDE_TEMPERATURE)
       call set_obs_def(obs, obs_def)  
       call add_obs_to_seq(seq, obs, time_obs, prev_obs, prev_time, first_obs)
    
-      call set_obs_def_error_variance(obs_def, 4.0_r8)
+      call set_obs_def_error_variance(obs_def, rawin_wind_error(obs_levels(lvl)))
```

Comment:
the err value in the obs_def keeps the previous value and since both U and V use the same error variance it doesn't need to be reset.  but if it would be clearer, that line could be duplicated again in the V section.  what do you think?

Reply:
or i could assign it to a w_err variable and set it in both places.  that might be the clearest?

Reply:
i added two comments about units - pascals vs hectopascals.  so i think all these issues have been addressed now.